purchases/src/test/java/com/revenuecat/purchases/PurchasesTest.kt

```diff
@@ -770,89 +1117,78 @@ class PurchasesTest {
         }
     }
 
+
+    // endregion
```

Comment:
Yeah, I think it'd be great to extract some of these blocks to different files. But that can be done in a different PR